lib/logstash/outputs/dynatrace.rb

```diff
@@ -44,80 +68,234 @@ class Dynatrace < LogStash::Outputs::Base
       # Disable SSL validation by setting :verify_mode OpenSSL::SSL::VERIFY_NONE
       config :ssl_verify_none, validate: :boolean, default: false
 
-      default :codec, 'json'
+      # Include headers in debug logs when HTTP errors occur. Headers include sensitive data such as API tokens.
+      config :debug_include_headers, validate: :boolean, default: false
 
-      attr_accessor :uri, :plugin_version
+      # Include body in debug logs when HTTP errors occur. Body may be large and include sensitive data.
+      config :debug_include_body, validate: :boolean, default: false
```

Comment:
New options not documented. I believe they may be helpful for us in the future helping users debug issues, but they log potentially sensitive data so I would keep them undocumented.

Reply:
Makes sense. Wondering if we should even allow logging headers (option above) or if that is something we'd want to avoid in all situations. :thinking:

Reply:
the upstream version of the http output plugin logged them by default. I decided to put them behind configs because I thought it might be helpful in the future. particularly if we ever support compression or other content encodings in the future.

Reply:
Alright :+1: 
Then let's keep it like that.